websites/B/Bots For Discord/dist/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+  "author": {
+    "name": "bris",
+    "id": "656165298818514946"
+  },
+  "service": "Bots For Discord",
+  "description": {
+    "en": "Bots for Discord a page listing the bots for Discord!"
```

Comment:
Use the official description;
`Find the perfect discord bot for your server on Bots For Discord, or list your bot for others to find.`

Reply:
Edited 👌